config.yaml

```diff
@@ -3,45 +3,84 @@ options:
     default: True
     type: boolean
     description: |
-      Limit requested identifiers (hostnames, IPs and OIDs) to first requester
+      This filter is enabled by default
```

Comment:
No need to state this, this is automatically rendered:
![image](https://github.com/canonical/tls-constraints-operator/assets/18486508/a7166d73-2e28-4932-ada4-a51137ba21e3)

This applies at a couple of places.

Reply:
removed